env0/resource_configuration_variable.go

```diff
@@ -130,6 +131,25 @@ func resourceConfigurationVariableCreate(ctx context.Context, d *schema.Resource
 	return nil
 }
 
+func getEnum(d *schema.ResourceData, value string) ([]string, diag.Diagnostics) {
+	var enumValues []interface{} = nil
+	var actualEnumValues []string = nil
+	if specified, ok := d.GetOk("enum"); ok {
+		enumValues = specified.([]interface{})
+		var valueExists = false
+		for _, val := range enumValues {
+			actualEnumValues = append(actualEnumValues, val.(string))
+			if val == value {
```

Comment:
the use of `val` AND `value` in the same spot is a little confusing 
can you make that `if enumValue == selectedValue` or something like that?

Reply:
42577a85